doc/Language/objects.pod6

```diff
@@ -1259,6 +1259,16 @@ say $a.^name;
 my $all-roles = 1 but (R1,R2);
 say $all-roles.^name; # OUTPUT: «Int+{R1,R2}␤»
 
+If the role supplies exactly one attribute, an initializer can be passed in
+parentheses:
+
+=begin code
+role Named {
+    has $.name;
+}
+my $hero = 1.Rat but Named('Remy');
```

Comment:
This causes 'make xtest' to fail - see comment on PR for syntax on how to check specific files (tests run much faster that way) and how to fix this one failure.

Reply:
Thanks for spotting the problem and the hint, dc42e2a fixes it.